{{cookiecutter.project_slug}}/_/deployment/python/deployment/database/base/deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: "postgres"
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  serviceName: "postgres"
+  template:
+    spec:
+      containers:
+      - name: postgresql
+        image: ${DOCKER_REGISTRY}/openshift/postgresql:9.6
+        imagePullPolicy: Always
+        ports:
+        - name: postgresql
+          containerPort: 5432
+        envFrom:
+        - secretRef:
+            name: "postgres"
```

Comment:
Can we make all references (name, port, etc.) either "postgres" (preferred) or "postgresql"?